packages/frontend/editor-ui/src/components/RunDataAi/utils.ts

```diff
@@ -257,6 +253,19 @@ export interface LogEntry {
 	runIndex: number;
 	runData: ITaskData;
 	consumedTokens: LlmTokenUsageData;
+	workflow: Workflow;
+	executionId: string;
+	execution: IRunExecutionData;
```

Comment:
Adding reference to workflow/execution data in each tree node to support different execution in the log tree